.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,26 @@
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - uses: SonarSource/sonarqube-scan-action@v4
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
+      - uses: amondnet/vercel-action@v20
+        with:
+          vercel-token: ${{ secrets.VERCEL_TOKEN }}
+          github-token: ${{ secrets.GH_TOKEN }}
+          vercel-args: '--prod'
+          vercel-org-id: ${{ secrets.ORG_ID_STAGING}}
+          vercel-project-id: ${{ secrets.PROJECT_ID_STAGING}}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)